### PR TITLE
added material ui, and updated design

### PR DESCRIPTION
**Pull request by noliver-ykc**
When you pull this make sure to move into the front end file and run "npm i" to install all necessary libraries. 
Ill continue onto the other pages with a similar design. 
If you have any stylistic ideas please let me know
I also created a logo and favicon

キャプチャ

### CODE REVIEW

**Comment by noliver-ykc** [comment]
*mobile margins will be added later on*

File: frontend/simple-mercari-web/src/App.css [hunk 1]
```diff
@@ -1,25 +1,47 @@
...
+#featured-items-img{
+  height: 100vh;
+  height: 100vh;
+  width: 50vw;
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+#accountForm {
```
Comment (mutokrm) [inline frontend/simple-mercari-web/src/App.css:12]: nit: I guess it's better to stay consistent with either camelCase or snake_case?

**Review by mutokrm — APPROVED** [review]
Added a small comment, but give you LGTM so you can merge when you fix it!

File: frontend/simple-mercari-web/src/App.css (hunk 1 above)
Comment (noliver-ykc) [reply frontend/simple-mercari-web/src/App.css:12]: ahhh そうだね. css is usually written with "-" but all the pre-installed css classes were in camelCase. As I improve the design ill slowly get rid of all the camelCase!

**Pull request merged**